docs/modules/plugins/pages/plugin-azure-cosmos-db.adoc

```diff
@@ -72,6 +72,9 @@ azure.cosmos-db.account.vividus.endpoint=https://vividus-cosmos.documents.azure.
 azure.cosmos-db.account.vividus.key=some key
 ----
 
+[IMPORTANT]
+In case if `account-key` was not provided in properties, will be used https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/identity/azure-identity#defaultazurecredential[DefaultAzureCredential mechanism] for authentication.
```

Comment:
1) put this info to the place where `key` is defined
2) reuse this partial: https://github.com/vividus-framework/vividus/blob/master/docs/modules/plugins/partials/azure-authentication.adoc (feel free to improve it if needed)